internal/logging/logging.go

```diff
@@ -0,0 +1,103 @@
+package logging
+
+import (
+	"fmt"
+	"io"
+	"sync"
+	"time"
+
+	"github.com/apex/log"
+)
+
+// Terminal colors
+const (
```

Comment:
I wonder if these should move to `styles.go`. That tends to be where we aggregate all the color-related tuning.